src/pages/index.js

```diff
@@ -122,6 +122,30 @@ export const homeQuery = graphql`
             type
           }
         }
+        ... on ContentfulBlockVisualList {
+          id
+          title
+          settingVariant
+          items {
```

Comment:
Add an id to the query for the cards just so we can use them ad keys when maping through the array.